ship-it-inc/ship-it-fend
# #167807195 set up react and webpack

#### What does this PR do?
It sets up react and webpack for the project
#### Description of Task to be completed?
- set up webpack
- set up react
- set up babel
- set up eslint
#### How should this be manually tested?
- Fetch the branch `ch-setup-react-webpack-167807195`
- Check out into the branch `ch-setup-react-webpack-167807195`
- run npm run dev
- The app should be started successfully
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167807195](https://www.pivotaltracker.com/story/show/167807195)
